activesupport/lib/active_support/test_case.rb

```diff
@@ -75,6 +75,11 @@ def test_order
     #     perform_service(param: 'no_exception')
     #   end
     def assert_nothing_raised(*args)
+      ActiveSupport::Deprecation.warn(<<-MSG.squish)
```

Comment:
Maybe you have to check presence of arguments. Now it's giving deprecation error every time even if you don't send any parameter to it.